src/main/resources/spring/camel-context.xml

```diff
@@ -38,6 +38,15 @@
             <apiProperty key="cors" value="true"/>
         </restConfiguration>
 
+        <rest path="/mappings">
+            <description>Mapping REST API</description>
+            <get uri="/flag-assessment/{flag}">
```

Comment:
With `>4 vNICs` flag, the `/api/xavier/mappings/flag-assessment/%3E4%20vNICs` endpoint failed returning `400 Bad Request`

Reply:
this problem should not happen again